webhook_to_fedora_messaging/models/service.py

```diff
@@ -0,0 +1,21 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from sqlalchemy import Column, Integer, UnicodeText, Boolean, ForeignKey
+from webhook_to_fedora_messaging.models.util import UUIDCreatableMixin, CreatableMixin
+
+from webhook_to_fedora_messaging.database import Base
+
+
+class Service(Base, UUIDCreatableMixin, CreatableMixin):
+    __tablename__ = "services"
+
+    id = Column("id", Integer, primary_key=True, nullable=False)
+    user_id = Column(
```

Comment:
We could have had an `m-n` relationship for users and services but something tells me that would be chaotic. Having just one person in charge of making changes feels like the way forward. If it is needed, people might elect to create a dedicated service FAS account.

The credentials of this dedicated service FAS account can be shared among a group of people if immediate changes need to be done for which the person in charge cannot be waited for. This, again, is a worst case scenario with a very small likelihood of happening.

Reply:
Hmm, I kinda dislike shared accounts, because it usually means accounts with a weak password and no OTP.
But OK, let's start with a single owner and we can change that later if needed.